hydra-cardano-api/src/Hydra/Cardano/Api/Tx.hs

```diff
@@ -17,6 +17,7 @@ import Cardano.Ledger.Core qualified as Ledger (Tx, hashScript)
 import Data.Bifunctor (bimap)
 import Data.Functor ((<&>))
 import Data.Map qualified as Map
+import Hydra.Cardano.Api.AlonzoEraOnwards (IsAlonzoEraOnwards (..))
```

Comment:
I don't see these imports as bad and can't quite follow your argument in the the PR description.

Reply:
I don't think they are bad. I think it's good that these type class files should have no dependencies other than cardano-api, and that this could be reflected in the library dependencies.